src/accelerate/commands/config/config_args.py

```diff
@@ -111,13 +111,25 @@ def __post_init__(self):
 
 @dataclass
 class ClusterConfig(BaseConfig):
-    num_processes: int
+    num_processes: int = None
     machine_rank: int = 0
     num_machines: int = 1
     main_process_ip: Optional[str] = None
     main_process_port: Optional[int] = None
     main_training_function: str = "main"
 
+    def __post_init__(self):
+        super().__post_init__()
+        assert self.num_processes is not None, "num_processes can't be None"
```

Comment:
Why add this default?

Reply:
I need to fix it a bit. Will work on it.